cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -45,7 +45,13 @@ func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error
 		if n, err = client.Nodes().Get(node.GetHostname(""), metav1.GetOptions{}); err != nil {
```

Comment:
This is my fault - `err != nil` is there incorrect. We are ending this loop on err.

Also test for `Labels == nil` should be there, because it can `nil` because it's declared as `omitempty`, so can be unmarshaled possibly to `nil`.